### ISSUE TITLE
End position missing when balloon released

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#224]
From #215:

>Release alerts: End position on sweater missing. @terracoda to provide examples.

Assigning to @terracoda to provide examples.

**Comment by terracoda** [comment]
@jessegreenberg, the new pattern for the release alerts is described in detail in the google doc [BASE: Grid Descriptions Simplified and Revised](https://docs.google.com/document/d/1WNVq5_-Hc9LSdeZVSn2edlwYwpHtGNT1ThFD9NXpZ5Q/edit#heading=h.4xn2gqwfg1t).
The alerts all follow this pattern:
- [Indication of the release.] + [Description of release movement.] + [Attractive state and location.] 

**(New) Release Alert 1: Balloon with neutral net charge**
- [+ Indication of release.] [No change in position]. [+ Attractive state and location].
 
**(New) Release Alert 2:  Balloon with negative net charge and simple attraction scenario**
- [+ Indication of release.] [Moved [[STRING:Velocity]] towards [[STRING:Destination location]].] [+ Attractive state and location].

**(New) Release Alert 3:  Balloon with negative net charge and a complex attraction scenario (multiple alerts)**
1. Similar alert in present progressive:
- [+ Indication of release.] [Moving extremely slowly towards [[STRING:Destination location]].] 

2. Release regular short alerts indicating movement in same direction is still happening: 
- [Still moving towards [[STRING:Destination location]].]

3. And, release a final alert once the balloon stops somewhere:
- [[STRING:Balloon name], [+ attractive state and location (with no capitalization)].

**Examples for neutral balloon release alerts with locations appended:**
- Yellow balloon released. No change in position. On left side of Play Area.
- Yellow balloon released. No change in position. Touching wall.
- Yellow balloon released. No change in position. Sticking to lower wall.
- Yellow balloon released. No change in position. On lower-left side of Play Area.

**Examples of simple balloon release alerts that fit the 4-point velocity model:**
- Yellow balloon released. Moved quickly to sweater. Sticking to right side of sweater.
- Yellow balloon released. No change in position. Sticking to right side of sweater.
- Yellow balloon released. Moved slowly to wall. Sticking to upper wall.

**Examples of more complex release alerts:**
- Yellow balloon released. Moving extremely slowly towards sweater. ...Still moving towards sweater. ...Still moving towards sweater. …Still moving towards sweater. ...Yellow balloon, sticking to right side of sweater.
- Green balloon released. Moving extremely slowly towards sweater and yellow balloon. ...Still moving towards sweater and yellow balloon. ...Still moving towards sweater and yellow balloon. ...Green balloon, sticking to lower-right arm of sweater.
- Green balloon released. Moving extremely slowly towards wall. …Still moving towards wall. …Still moving towards wall. ...Green balloon, sticking to wall.

**Comment by terracoda** [comment]
@jessegreenberg, cognitive walk-throughs with release examples are also provided in the google sheet, [BASE: CogWT](https://docs.google.com/spreadsheets/d/1fZq21j65jfJhcsA22pasuBA00kM-cGWxsWDsogwhC64/edit#gid=0). Walk-throughs 7,8,9 and 10 demo releases.

**Comment by jessegreenberg** [comment]
@terracoda we have balloon position at the end of release alerts now.  Tested each of the examples listed in https://github.com/phetsims/balloons-and-static-electricity/issues/224#issuecomment-285338660, it sounds correct (thought not exactly as listed because we have since changed the wording in many places).
